backend/app/routes/admin/getAdminList.js

```diff
@@ -0,0 +1,34 @@
+const Admin = require('../../models/Admin');
+
+function getPropsFromAdminData(a) {
+  let json = '{';
+  if (!a.includes('_id')) json += '"_id":0,';
+  a.forEach(e => json += `"${e}": "$${e}",`);
+  json = json.replace(/.$/, '}')
+  return JSON.parse(json);
+}
+
+module.exports = async (req, res, next) => {
+  const options = req.query;
+  const propsFromAdminData = ['email', 'firstName'];
+  const limitOfPagination = 3;
+  let { pageNumber, isSuperAdmin } = options;

```

Comment:
there will be 2 query parameters both optional
type - ['superAdmin','admin']
page - int representing a page number
if type param is absent then simply fetch all records. just change make the appropriate changes to variable names etc. Make a function to prepare the pipeline. Pass the type and page as arguments to it